index.html

```diff
@@ -0,0 +1,235 @@
+      <div class="project-image">
+        <img src="/assets/images/mobile/project3.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project4.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project5.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+  </section>
+
+
+  <!-- The About me Section -->
+  <section class="about-me">
+    <h1 class="about-title">About</h1>
+    <p class="about-desc">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+    </p>
+    <button class="intro-button resume-btn">
+      Get My CV
+      <img src="assets/images/mobile/download.svg" alt="download-icon" class="resume-icon">
+    </button>
+  </section>
+
+  <section class="technologies">
+    <h2 class="tech-title">Languages</h2>
+    <ul class="stack-list">
+      <li class="stack-item">Javascript</li>
+      <li class="stack-item">Ruby</li>
+      <li class="stack-item">HTML</li>
+      <li class="stack-item">CSS</li>
+    </ul>
+  </section>
+
+  <section class="frameworks">
+    <h2 class="framework-title">Frameworks</h2>
```

Comment:
- [ ] Make sure your design sticks to the template design you chose; please check the alignment for the about me section of your template design.

***Your design***
![Screenshot from 2022-12-03 18-43-48](https://user-images.githubusercontent.com/65129228/205450373-574b71c4-7aeb-44b3-8b62-47c507316d02.png)